IdentityCore/src/cache/accessor/MSIDAccountCredentialCache.m

```diff
@@ -465,4 +465,20 @@ - (nullable MSIDAppMetadataCacheItem *)getAppMetadata:(nonnull MSIDAppMetadataCa
     return [_dataSource appMetadataWithKey:key serializer:_serializer context:context error:error];
 }
 
+- (BOOL)removeAppMetadata:(nonnull MSIDAppMetadataCacheItem *)appMetadata
+                  context:(nullable id<MSIDRequestContext>)context
+                    error:(NSError * _Nullable * _Nullable)error
+{
+    assert(appMetadata);
+    
+    MSID_LOG_VERBOSE(context, @"(Default cache) Removing app metadata with clientId %@", appMetadata.clientId);
+    MSID_LOG_VERBOSE_PII(context, @"(Default cache) Removing app metadata with clientId %@, environment %@", appMetadata.clientId, appMetadata.environment);
```

Comment:
environment is not PII, we can log it in normal log

Reply:
done